doc/media/openage/modpacks.md

```diff
@@ -0,0 +1,176 @@
+# Modpacks
+
+Input assets for openage are organized in packages that we call **modpacks**.
+Everything you need to know about them is described in this document.
+
+
+## What does a modpack contain?
+
+Modpacks contain anything that is considered an input for the engine, e.g.
+gamedata, graphics, sounds, scripts, translations or a combination of all
+of those. There are no limits on their scope. A modpack can contain
+anything from minor enhancements to standalone games.
+
+Apart from game-related data, modpacks store configuration data for loading
+them into the engine. Most of this data is located in the
+**[modpack definition file](###modpack-definition-file)**. Modpacks can define
+their interaction with other modpacks by specifying **dependencies** and
+**conflicts**.
+
+The organization of files inside the modpack can be decided by its creators.
+openage does not enforce a specific file structure, although a "sensible"
+structure is of course recommended. The only file that is bound to some requirements
+is the modpack definition file (see its dedicated section for details).
+
+
+### Modpack Definition File
+
+The modpack definition file acts as the header file for the modpack. It is
+the only mandatory file in the modpack. The definition file must be placed
+in the modpack's root folder under the filename `modpack.nfo`.
```

Comment:
maybe we should think of a better file extension :D 
ideas: `.oam, .mod, .oamod, .openagemod, .midx, ...`

Reply:
Everyone was happy with `.nfo` so far, so we don't need to get fancy (yet) :D